packages/orchestration/src/orchestration-stream-chunk-response.ts

```diff
@@ -42,6 +43,17 @@ export class OrchestrationStreamChunkResponse {
     )?.delta.content;
   }
 
+  /**
+   * Parses the chunk response and returns the tool call chunks generated by the model.
+   * @param choiceIndex - The index of the choice to parse.
+   * @returns The message tool call chunks.
+   */
+  getDeltaToolCallChunks(choiceIndex = 0): ToolCallChunk[] | undefined {
+    return this.getChoices()?.find(
+      (c: LlmChoiceStreaming) => c.index === choiceIndex
+    )?.delta.tool_calls;
+  }
+
   private getChoices(): LlmChoiceStreaming[] | undefined {
```

Comment:
In non-streaming case, we implemented `findChoiceByIndex()` to improve code reuse. We can do the same thing here.